src/index.js

```diff
@@ -1,17 +1,11 @@
 import React from 'react';
 import ReactDOM from 'react-dom/client';
-import './index.css';
 import App from './App';
-import reportWebVitals from './reportWebVitals';
+import './App.css';
 
 const root = ReactDOM.createRoot(document.getElementById('root'));
 root.render(
   <React.StrictMode>
```

Comment:
- [ ] Your `index` should have 2 `<Route>`s, one for the BookList component and the other for the Categories component
- [ ] The BookList component should be the default page, please implement these changes so your project respects the guidelines
```
<Route exact path="/">
      <BookList />
</Route>
```

Reply:
I made these two components in the app.js module already. I don't know why it doesn't show you, brother.